graftm/create.py

```diff
@@ -181,14 +182,18 @@ def _compile(self, base, refpkg, hmm, diamond_database_file, prefix):
         shutil.copyfile(hmm, os.path.join(gpkg, hmm_file_in_gpkg))
         
         diamond_database_file_in_gpkg = os.path.basename(diamond_database_file)
+        taxonomy_file_in_refpkg       = os.path.join(refpkg, tax_info)
         shutil.copyfile(diamond_database_file, os.path.join(gpkg, diamond_database_file_in_gpkg))
         
         contents = {"version": 2,
-            "aln_hmm": hmm_file_in_gpkg,
-            "search_hmm": [hmm_file_in_gpkg],
-            "rfpkg": refpkg,
-            "TC":False,
-            "diamond_database": diamond_database_file_in_gpkg}
+                    "aln_hmm": hmm_file_in_gpkg,
+                    "search_hmm": [hmm_file_in_gpkg],
+                    "rfpkg": refpkg,
+                    "TC":False,
+                    "diamond_database": diamond_database_file_in_gpkg,
+                    "range": max_range,
+                    "tax_info": taxonomy_file_in_refpkg}
```

Comment:
perhaps I missed it, but as far as I can see this is not used at all - is it needed?